### PR TITLE
Added lego, playmobil.de and updated ebay-de

### PR DESCRIPTION
**Pull request by mal-tee**
Please note that both numbers of the playmobil entry are general numbers and not privacy-related.
Is this correct, do we use and fall back to these numbers even if there is a privacy-related email address?

### CODE REVIEW

File: companies/playmobil-de.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "entertainment"
+    ],
+    "name": "geobra Brandstätter Stiftung & Co. KG",
+    "address": "geobra Brandstätter Stiftung & Co. KG\nBrandstätterstraße 2 - 10\nD - 90513 Zirndorf\nDeutschland",
+    "phone": "+49 911 96660",
+    "fax": "+49 911 96661120",
+    "email": "dsb@playmobil.de",
+    "web": "https://www.playmobil.de",
+    "sources": [
+        "https://www.playmobil.de/startseite"
```
Comment (baltpeter) [inline companies/playmobil-de.json:17]: If you middleclick the link, you get the actual URL. :D 

```suggestion
        "https://www.playmobil.de/content/privacy_policy/PRIVACY_POLICY.html"
```

File: companies/playmobil-de.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "playmobil-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce",
+        "entertainment"
+    ],
+    "name": "geobra Brandstätter Stiftung & Co. KG",
```
Comment (baltpeter) [inline companies/playmobil-de.json:10]: Shouldn't there be a `runs` for `*playmobil*`? :D

File: companies/playmobil-de.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "address": "geobra Brandstätter Stiftung & Co. KG\nBrandstätterstraße 2 - 10\nD - 90513 Zirndorf\nDeutschland",
```
Comment (baltpeter) [inline companies/playmobil-de.json:11]:
```suggestion
    "address": "Brandstätterstraße 2 - 10\n90513 Zirndorf\nDeutschland",
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks!

File: companies/playmobil-de.json (hunk 2 above)
Comment (mal-tee) [reply companies/playmobil-de.json:10]: Oh yes, ofc, great

**Comment by mal-tee** [comment]
Please merge  datenanfragen/company-json-generator#8 so this doesn't happen again :D

**Review by baltpeter — APPROVED** [review]

**Pull request merged**